### ISSUE TITLE
Are QSPR/ML submissions accepted and if so, how should they be formatted?

### ISSUE THREAD

**Issue by IamDavyG** [samplchallenges/SAMPL8#22]
I am looking at the submission template and it has "QSPR/ML" as a category but it also wants relative free energy values so I'm not sure if QSPR/ML models are accepted in the SAMPL8 pKa challenge? Looking back at the SAMPL7 Chemicalize reference set, it seems it just uses the output from ChemAxon's pKa calculator without conversion but I want to confirm if the format is exactly the same, provided these models are accepted? I'd rather not get it wrong and convert predicted LogP values to relative free energies like in SAMPL7 LogP challenge.

**Comment by davidlmobley** [comment]
I'm slightly confused as to whether you asking whether pKa predictions need to be in terms of free energies, or whether logD predictions need to be free energies. Can you clarify?

Thanks.

**Comment by IamDavyG** [comment]
I am just asking about the pKa challenge for now as I'm slightly confused about [this reference submission](https://github.com/samplchallenges/SAMPL7/blob/master/physical_property/pKa/analysis/relative_microstate_free_energy_predictions/pKa_REF00_Chemaxon_Chemicalize_1.csv) that just used pKa values instead of free energies. I know because I could reproduce the values from ChemAxon's pKa calculator. Can I submit pKa values in that format for a QSPR/ML model or was it just a one off exception for the reference submission?

**Comment by davidlmobley** [comment]
Thanks for clarifying. Indeed, we DO need relative free energies between microstates, and not pKa values. It looks like @bergazin may have deposited an incorrectly formatted reference file for some reason. 

If this would preclude you participating (e.g. if you only have macro pKa predictions and not microstate predictions, for example) let me know and we can talk about whether there are other options. Unfortunately it's nearly impossible for us to analyze the results of methods which only predict macro pKa's (because if there are multiple transitions, it can become impossible to determine which transitions a given pKa corresponds to) but we could probably still serve as a curator to verify such predictions were done blindly.

In other words, if you can't participate if you're required to submit relative free energies between microstates, let me know and I'll see whether there are other people who have the same problem and, if so, potentially run an additional challenge where we accept macro pKa's without microstate free energies. We would not evaluate this challenge, but would be able to verify that predictions were made blindly.
